src/hooks/useNotificationTypes.tsx

```diff
@@ -281,7 +281,7 @@ export const notificationTypes: NotificationTypeConfig[] = [
       const conditionalOrdersExpirationDate = new Date('2024-06-01T23:59:59');
       const fokDeprecationExpirationDate = new Date('2024-07-01T23:59:59');
       const isolatedMarginLiveExpirationDate = new Date('2024-07-12T23:59:59');
-      const stakingLiveExpirationDate = new Date('2024-08-24T23:59:59');
+      const stakingLiveExpirationDate = new Date('2024-07-24T23:59:59');
```

Comment:
ok this was supposed to be a month from when I [merged](https://github.com/dydxprotocol/v4-web/pull/708/files#diff-9f5e939dc0008478c54630b20e47a834282e09041fc3a4133da1ff5ab0d239bdR292) it but I can't count apparently